dev/tests/phpunit/bootstrap-common.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * WP Rig common tests bootstrap script.
+ *
+ * @package wp_rig
+ */
+
+// Disable xdebug backtrace.
+if ( function_exists( 'xdebug_disable' ) ) {
+	xdebug_disable();
```

Comment:
This will completely disable xdebug? How do you generate code coverage?

I'd disable xdebug in the travis, so that it's faster there...